src/translations/sqlb_es_ES.ts

```diff
@@ -4173,32 +4173,32 @@ Si responde no se intentarán importar los datos del archivo SQL en la base de d
     <message>
         <location filename="../MainWindow.cpp" line="2075"/>
         <source>The statements in the tab &apos;%1&apos; are still executing. Closing the tab will stop the execution. This might leave the database in an inconsistent state. Are you sure you want to close the tab?</source>
-        <translation type="unfinished"></translation>
+        <translation>Las sentencias en la pestaña &apos;%1&apos; todavía se están ejecutando. Al cerrar la pestaña se detendrá la ejecución. Esto puede dejar la base de datos en un estado inconsistente. ¿Está seguro de que quiere cerrar la pestaña?</translation>
     </message>
     <message>
         <location filename="../MainWindow.cpp" line="2704"/>
         <source>DB file &apos;%1&apos; could not be opened</source>
-        <translation type="unfinished"></translation>
+        <translation>El archivo de base de datos &apos;%1&apos; no se pudo abrir</translation>
     </message>
     <message>
         <location filename="../MainWindow.cpp" line="2841"/>
         <source>This project file is using an old file format because it was created using DB Browser for SQLite version 3.10 or lower. Loading this file format is no longer fully supported. If you want to load it completely, please use DB Browser for SQLite version 3.12 to convert it to the new file format.</source>
-        <translation type="unfinished"></translation>
+        <translation>Este proyecto está usando un formato de archivo antiguo porque fue creado usando DB Browser for SQLite version 3.10 o menor. La carga de este formato de archivo no está soportado en su totalidad. Si quiere cargarlo completamente, por favor, use DB Browser for SQLite version 3.12 para convertirlo al nuevo formato.</translation>
```

Comment:
¿Debería ser "versión" aquí en vez de _version_?

Reply:
No lo he traducido porque he tomado DB Browser for SQLite version 3.10 como un todo, aunque debería haberlo entrcomillado.

Lo voy a cambiar.